src/utils/utils-vega.js

```diff
@@ -190,13 +196,23 @@ export function createVegaAttrMixin(
           }
           domainVals = layerObj[domainGetterFunc]()
         }
-        if (capAttrObj.type === "ordinal") {
+        if (scaleType === "ordinal") {
           ordScale.domain(domainVals).range(capAttrObj.range)
           // if color range is not in domain, it's an Other item
           rtnVal =
             domainVals.indexOf(input) === -1
               ? ordScale("Other")
               : ordScale(input)
+        } else if (scaleType === "nominal") {
+          let formattedInput = input
+          if (domainVals.every(v => typeof v === "boolean")) {
+            formattedInput = Boolean(input)
+          }
+          const indexOfVal = domainVals.indexOf(formattedInput)
```

Comment:
you are essentially conducting your own `ordinal` scale lookup here. Any reason you're doing this manually rather than using the d3 ordinal scale?

Reply:
Nope good call, maybe this can even just fall into the same block as ordinal

Reply:
yeah, it probably can (other than the boolean type conversion -- tho it's possible that could just work)

Reply:
Agh, yeah it needs this conversion to handle getting 0/1 as an input when the domain is boolean, but otherwise I can roll it in with the previous condition

Reply:
I take it back, 0/1 does work with a d3 ordinal scale of false/true! I'm going to keep this as a separate case though, because it doesn't handle "Other" the same as the "ordinal" vega scale type

Reply:
I double take it back, this works in newer versions of d3 but not ours, i'm going to keep boolean logic here as well

Reply:
Ok, final state is that "nominal" scale is its own block, it does the manual boolean conversion if the domain is all booleans, but it uses the d3 ordinal scale